google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_datacatalog_tag_factory.py

```diff
@@ -14,15 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+
 from google.cloud import datacatalog
 
 from google.datacatalog_connectors.commons import prepare
+from google.datacatalog_connectors.commons.utils import region_tag_helper
```

Comment:
```suggestion
from google.datacatalog_connectors.commons import utils
```
Less verbose... WDYT? 🤔